recipe/meta.yaml

```diff
@@ -33,31 +33,29 @@ requirements:
     # Sync deps with https://github.com/aimhubio/aim/blob/v3.16.0/setup.py
     - python
     - aim-ui =={{ version }}
-    - aimrecords ==0.0.7
-    - aimrocks ==0.4.0
+    - aimrocks ==0.5.1
+    - khash ==0.5.0a5
     - cachetools >=4.0.0
     - click >=7.0
     - cryptography >=3.0
     - filelock >=3.3.0,<4
     - numpy >=1.12.0,<2
     - psutil >=5.6.7
-    - py3nvml >=0.2.5
     - RestrictedPython >=5.1
     - tqdm >=4.20.0
-    - aiofiles >=0.5.0
     - alembic >=1.5.0,<2
     - fastapi >=0.69.0,<1
     - jinja2 >=2.10.0,<4
-    - pytz >=2019.1
     - SQLAlchemy >=1.4.1,<2
     - uvicorn >=0.12.0,<1
     - Pillow >=8.0.0
-    - protobuf >=3.9.2,<5
     - packaging >=15.0
-    - python-dateutil
+    - websockets
+    - watchdog
     - requests
-    - grpcio >=1.42.0
-    - analytics-python >=2.2.2
+    - segment-analytics-python
```

Comment:
@alberttorosyan `segment-analytics-python` is no longer in setup.py and is optional?